src/framework/anim/evaluator/anim-track.js

```diff
@@ -110,4 +110,9 @@ class AnimTrack {
     }
 }
 
+/**
+ * This AnimTrack can be used as a placeholder track when creating a state graph before having all associated animation data available.
+ */
+AnimTrack.EMPTY = new AnimTrack('empty', Number.MAX_VALUE, [], [], []);
```

Comment:
could this please use the same format as for example static colors:

    static WHITE = Object.freeze(new Color(1, 1, 1, 1));

Reply:
Technically it would require a recursive/deep freeze, since the arrays are still mutable